pyfstat/mcmc_based_searches.py

```diff
@@ -1804,9 +1804,15 @@ def get_summary_stats(self):
             d[k] = {}
             d[k]["mean"] = np.mean(s)
             d[k]["std"] = np.std(s)
-            d[k]["lower90"], d[k]["median"], d[k]["upper90"] = np.quantile(
-                s, [0.05, 0.5, 0.95]
-            )
+            (
+                d[k]["lower99"],
+                d[k]["lower90"],
+                d[k]["lower50"],
+                d[k]["median"],
+                d[k]["upper50"],
+                d[k]["upper90"],
+                d[k]["upper99"],
+            ) = np.quantile(s, [0.005, 0.05, 0.25, 0.5, 0.75, 0.95, 0.995])
```

Comment:
I'm not sure if I get this: Why e.g. `lower99` goes with 0.005?

Reply:
symmetric 99% interval = 99% contained around the median -> [0.005,0.995] with half a percent left out either side

Reply:
Of course, that makes sense.